Source/Core/AudioCommon/CubebUtils.cpp

```diff
@@ -82,3 +91,146 @@ std::shared_ptr<cubeb> CubebUtils::GetContext()
   weak = shared = {ctx, DestroyContext};
   return shared;
 }
+
+std::vector<std::pair<std::string, std::string>> ListInputDevices()
```

Comment:
These strings ultimately come from a cubeb context which is a static.  Are we able to reference these names as a `string_view` instead?

Reply:
I don't think that's safe since these strings come from a collection that I need to destroy at the end of the function. I also couldn't find examples in cubeb source code where they're using `string_view` to reference them.